birdhouse/config/thredds/default.env

```diff
@@ -5,11 +5,30 @@ export THREDDS_VERSION=4.6.18-unidata-2022-01
 export THREDDS_DOCKER=pavics/thredds-docker
 export THREDDS_IMAGE="${THREDDS_DOCKER}:${THREDDS_VERSION}"
 export THREDDS_ORGANIZATION="Birdhouse"
+export THREDDS_ADDITIONAL_CATALOG=""
+
+# All default values for data locations are set to ensure backward compatibility
+export THREDDS_DATASET_LOCATION_ON_CONTAINER='/pavics-ncml'
+export THREDDS_SERVICE_DATA_LOCATION_ON_CONTAINER='/pavics-data'
```

Comment:
Might as well say `/pavics-ncml` and `/pavics-data` are for backward compatibilty because their defaut value are also "funny".